Makefile

```diff
@@ -143,17 +146,25 @@ docs_doxygen_only:
 docs_external:
 	$(Q)make -C ../libxlsxwriter.github.io release
 
+pkg-config:
```

Comment:
I added these two files to `.gitignore`. The `*-e` file is macOS specific as explained [here](https://unix.stackexchange.com/questions/13711/differences-between-sed-on-mac-osx-and-other-standard-sed/131940#131940). I wrote all the `sed` commands with macOS and Linux compatibility in mind.

Reply:
> I added these two files to `.gitignore`.

I don't like that approach. It leaves too many files in the root dir. As I suggested above it would probably be better to just pipe the translated template output directly to the target file. Something like this:

```
sed -e 's|@PREFIX@|$(PREFIX)|g'  -e 's|@VERSION@|$(VERSION)|g' dev/release/pkg-config.txt > $(DESTDIR)$(PREFIX)/lib/xlsxwriter.pc
```